cnf-tests/.konflux/rpms.lock.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Maybe its worth adding a readme file / short script in the meanwhile to show how to update this file ?

Reply:
Generating it can be done following the steps in the docs, adding a link to that is handy and a good idea. I see the point that updating RPMs deps is a concern but this file shouldn't never be updated manually or else runs will very likely fail if updates were missed for the input file and the repo. ADd to that that updating the repo file will need adjustments to the activation key reference in konflux instance and the one used to locally generate the lockfile.

Reply:
I have added a readme  file in the last commit, please consider this will be adjusted once we have settled on an activation key.